.env.sample

```diff
@@ -7,4 +7,5 @@ SERVER_MAIL= email address
 google_client=string from google
 google_secret=string from google
 facebook_client=string from facebook
-facebook_secret=string from facebook
\ No newline at end of file
+facebook_secret=string from facebook
+APP_URL=heroku_link
```

Comment:
Mae this application URL instead of Heroku link

Reply:
ok

Reply:
Will that be all? I want to know so I can fix them all in one commit